app/src/main/kotlin/org/open/smsforwarder/ui/history/adapter/HistoryViewHolder.kt

```diff
@@ -12,6 +12,10 @@ class HistoryViewHolder(
 
     fun bind(item: HistoryUI) = with(binding) {
         val context = itemView.context
+        dateLabel.contentDescription = context.getString(
+            R.string.forwarding_history_item_content_description,
+            item.id.toString()
+        ) + context.getString(R.string.forwarding_history_item_date)
```

Comment:
why didn't you make 1 place holder here?
You can all pas in one placeholder without need to "+" (concatenate) the string